adam/visualization/make_scenes.py

```diff
@@ -111,21 +111,33 @@ def main(params: Parameters) -> None:
         logging.info("SCALE: %s -> %s", model_name, scale.__str__())
 
     for i, scene_elements in enumerate(SceneCreator.create_scenes([make_curriculum()])):
-
+        if i < 1:
```

Comment:
What is this for?

Reply:
Updating to make the _specific_ selection of a scene for testing purposes explicit in the params file

Reply:
I think I need more context to understand.

Reply:
I broke a bunch of things by changing back to rendering separate objects for sub-objects

The new params option allows one to select a specific numbered scene if they don't want to bother scrolling through a bunch of them. I just pushed an update to fix the back-up option of rendering a basic Geon if there isn't a specific model available. So at the moment, rendering a table is still broken (haven't separated the model), but at least it doesn't crash the program. 

The current working example of a separated model is the chair.